features/Kiesrecht.feature

```diff
@@ -42,23 +105,78 @@ Scenario: de ingeschreven persoon is uitgesloten van kiesrecht
 Scenario: de ingeschreven persoon is niet uitgesloten van kiesrecht
 	Gegeven op de PL van een ingeschreven persoon is er geen categorie 13, element 38.10 (= persoon is niet uitgesloten)

```

Comment:
Moet de tekst '(= persoon is niet uitgesloten)' hier niet weg? Want we zeggen toch dat als element 38.10 niet aanwezig is dat wij niet met zekerheid kunnen zeggen of de persoon kiesrecht heeft of niet?